gulp/build-tasks/modules.js

```diff
@@ -5,7 +5,7 @@ const ExtractTextPlugin = require('extract-text-webpack-plugin');
 module.exports = function (gulp, plugins, config, gulpWebpack, webpack, path) {
   return function () {
     //init is a global component the scripts inside of this are loaded on every page. For example - loader, license
-    let modules = ['loader', 'slider', 'autocomplete', 'pagination', 'data', 'misc', 'social-feed'];
+    let modules = ['loader', 'slider', 'autocomplete', 'pagination', 'data', 'misc', 'social-feed', 'jquery.status', 'jquery.history'];
```

Comment:
Accepted. But currently lets do exact transfer of functionality. Since we are not looking to change functionality in the current sprints I left the names as it is in SWE2